go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/AndreasBriese/bbloom v0.0.0-20190825152654-46b345b51c96 // indirect
 	github.com/alecthomas/jsonschema v0.0.0-20191017121752-4bb6e3fae4f2
 	github.com/caarlos0/spin v1.1.0
+	github.com/cenkalti/backoff v2.2.1+incompatible
```

Comment:
Is this +incompatible something to worry about?

Reply:
I don't think so.
@sanderpick, btw, maybe would be better to use `	github.com/cenkalti/backoff/v4`

Reply:
Huh, that is the one I used with `go get`, maybe it downgraded due to some incompatibility. `go.sum` has:

```
github.com/cenkalti/backoff v2.2.1+incompatible h1:tNowT99t7UNflLxfYYSlKYsBpXdEet03Pg2g16Swow4=
github.com/cenkalti/backoff v2.2.1+incompatible/go.mod h1:90ReRw6GdpyfrHakVjL/QHaoyV4aDUVVkXQJJJ3NXXM=
github.com/cenkalti/backoff/v3 v3.0.0 h1:ske+9nBpD9qZsTBoF41nW5L+AIuFBKMeze18XQ3eG1c=
github.com/cenkalti/backoff/v3 v3.0.0/go.mod h1:cIeZDE3IrqwwJl6VUwCN6trj1oXrTS4rc0ij+ULvLYs=
```

Reply:
Oh, maybe IDE picked up the wrong import path from local packages. Will look

Reply:
@sanderpick, in buckets/local/watch.go import section should say `"github.com/cenkalti/backoff/v4"`

Reply:
Hmm, no, `mod tidy` seems to revert to the older version, weird